components/gallery/GalleryItem.vue

```diff
@@ -177,6 +189,9 @@ onMounted(() => {
   })
 })
 
+// change when there is api for unloackable
```

Comment:
??

Reply:
I've been requested by @yangwao to make a draft PR for this although there is no data source for unlock-able property of NFT